src/mlnet/Templates/Console/ModelBuilder.tt

```diff
@@ -188,18 +188,18 @@ else{#>
         {
             var L1 = crossValidationResults.Select(r => r.Metrics.MeanAbsoluteError);
             var L2 = crossValidationResults.Select(r => r.Metrics.MeanSquaredError);
-            var RMS = crossValidationResults.Select(r => r.Metrics.MeanAbsoluteError);
+            var RMS = crossValidationResults.Select(r => r.Metrics.RootMeanSquaredError);
             var lossFunction = crossValidationResults.Select(r => r.Metrics.LossFunction);
             var R2 = crossValidationResults.Select(r => r.Metrics.RSquared);
```

Comment:
Something is wrong in here. Looks like you have changed tt file but no corresponding change in .cs file.